.github/workflows/backwards-compatibility-review.yml

```
@@ -0,0 +1,67 @@
+name: Backwards Compatibility PR Review
+
+on:
+  pull_request:
+    types: [opened, synchronize, ready_for_review, reopened]
+
+concurrency:
+  group: bc-review-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  review-with-tracking:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: PR Review with Progress Tracking
+        uses: anthropics/claude-code-action@v1
+        with:
+          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+
+          # Enable progress tracking
+          track_progress: true
+
+          # Your custom review instructions
+          prompt: |
+            REPO: ${{ github.repository }}
+            PR NUMBER: ${{ github.event.pull_request.number }}
+
+            PRIMARY OBJECTIVE: Backwards Compatibility (BC) Review for WooCommerce/WordPress
+
+            Only report on BC risks and required deprecations. Ignore styling, formatting, comments, and pure test/doc changes unless they mask a BC concern.
+
+            Output format (strict):
+            - Summary: Overall risk [None|Low|Medium|High] and a 2 to 4 sentence rationale.
+            - Testing Focus: targeted regression areas (e.g., checkout, gateways, subscriptions) with concrete steps.
+            - Confidence: a percentage and one-sentence justification.
+
+            Scope to analyze:
```

What do you think about:
```suggestion
            Scope to analyze (but not limited to):
```
to prevent accidental scope limitation to just these areas?

---

Thanks. I applied your suggestion.